### DISCUSSION TITLE
CPU hung . First time running transcibe

### DISCUSSION THREAD

**Discussion by arunpatil1988** [openai/whisper#1231] [Q&A]
Hi Guys - This is first time i installed whisper and all the related components.  I gave the below command and it is stuck

whisper /home/abcUser/file_2135900566_3093552530792048272.wav --model medium
100%|█████████████████████████████████████| 1.42G/1.42G [01:04<00:00, 23.8MiB/s]
/usr/local/lib/python3.10/site-packages/whisper/transcribe.py:114: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")
Detecting language using up to the first 30 seconds. Use `--language` to specify the language
Detected language: English

It is stuck at this point and nothing happening . Is it common and below is the system details. Let me know if its good enough to run whisper.

This is the system hardware details.

Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                2
On-line CPU(s) list:   0,1
Thread(s) per core:    1
Core(s) per socket:    1
Socket(s):             2
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 85
Model name:            Intel(R) Xeon(R) Gold 6154 CPU @ 3.00GHz
Stepping:              4
CPU MHz:               2992.968
BogoMIPS:              5985.93
Hypervisor vendor:     VMware
Virtualization type:   full
L1d cache:             32K
L1i cache:             32K
L2 cache:              1024K
L3 cache:              25344K
NUMA node0 CPU(s):     0,1

**Comment by arunpatil1988** [comment]
nvm - i killed the session and reran it again , was able to get the transcription. However on rerunning it was a lot faster, still taking on average 5 seconds. how do i speed this up.

**Reply by bangyuanzhu** [reply]
The medium model size costs more time. For English transcription, the small size is enough to get an acceptable WER. 

Another way is to move the model implement on a more powerful CPU/GPU
